main.tf

```
@@ -79,6 +79,7 @@ module "storage" {
   location            = azurerm_resource_group.default.location
   create_queue        = !var.use_internal_queue
   deletion_protection = var.deletion_protection
+  blob_container_name = "wandb"
```

```suggestion
  blob_container_name = "wandb"
```
Should this be hardcoded?

---

Oh, good call.

---

it was hardcoded before within the `storage` module. I changed it so the `storage` module accepts it as a variable now, so we could pass in a custom name from `secure_storage_connector`

are you saying we should make it a variable at the main level too?